src/Configuration/ValueObject/SnowflakeDatabaseConfig.php

```diff
@@ -55,4 +55,12 @@ public function getWarehouse(): string
         }
         return $this->warehouse;
     }
+
+    public function getPassword(bool $escapeSemicolon = false): string
```

Comment:
nerozumím tomu proč je to na parametr, proč se to nedělá vždy?

Reply:
jasně, no protože se to musí escapovat jen do odbc a do snowsql driveru:
https://github.com/keboola/db-extractor-snowflake/blob/master/src/Extractor/SnowsqlExportAdapter.php#L279
se to zase escapovat nesmí

Reply:
všechny hesla z testu - https://github.com/keboola/db-extractor-snowflake/pull/123/files#diff-9968fb44882794abc9b4ba8dfdcf8bc4e6966e8682ee6c3a0acc8cdb9161b5a1R30

jsem zkoušel oproti snflk databázi a projdou jak s odbc tak se snowsql

Reply:
oh, fock